src/obb/server/routes.clj

```diff
@@ -0,0 +1,13 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.routes
+  "Specifices the routes and associates handlers"
+  (:require
+    [compojure.core :as compojure :refer [GET]]
+    [compojure.route :as route]
+    [obb.server.handlers.index :as index]))
+
+(compojure/defroutes public-routes
+  "The routes available to be served, that don't need auth"
+  (GET "/" request (index/handler request))
+  (route/not-found "No such page."))
```

Comment:
I actually came across this yesterday because I set up a new Compojure app as well and noticed that template brings that by default. The main reason why I care about this stuff is that we often neglect the semantics that HTTP already provides. I'd expect that a client would never have to look into the body of a 404 response.

But that's OK, I know I am too picky with this stuff. The ultimate proof is that sometimes not even the frameworks implement these things as I think they should be. :p